src/functions/function_utils.cpp

```diff
@@ -47,8 +51,8 @@ namespace mrcpp {
  * integral is conserved with respect to the integration limits.
  *
  */
-template <int D> std::shared_ptr<GaussExp<D>> function_utils::make_gaussian_periodic(const Gaussian<D> &inp, const std::array<double, D> &period, double nStdDev) {
-    auto gauss_exp = std::make_shared<GaussExp<D>>();
+template <int D> GaussExp<D> function_utils::periodify(const Gaussian<D> &inp, const std::array<double, D> &period, double nStdDev) {
```

Comment:
Add `#include "functions/function_utils.h"` in  `api/Gaussians`. We need this in `mrchem`.

Reply:
Actually, perhaps we should just re-introduce it as a member function of the Gaussian? I like this syntax better:
```
auto periodic_exp = gfunc.periodify(period);
```

Reply:
I agree